Makefile

```diff
@@ -290,6 +290,10 @@ clang-format:
 LIVENESS_PROBE ?= true
 .PHONY: minikube-deploy
 minikube-deploy: minikube-start gadget-container kubectl-gadget
+	# Remove all resources created by Inspektor Gadget
+	./kubectl-gadget undeploy || true
+	# Remove the image from Minikube
+	$(MINIKUBE) image rm $(CONTAINER_REPO):$(IMAGE_TAG) || true
```

Comment:
Since I am only removing the image from the `minikube` it would just be loaded again without any network involved. It is just that the copying from host to minikube is what I want to be newest all the time. For example, in case you have an existing image ` make minikube-deploy -o gadget-container` still works fine.  Am I missing something?